src/window/window_wrapper/keyboard_manager.rs

```diff
@@ -8,14 +8,18 @@ use crate::channel_utils::LoggingTx;
 use crate::settings::SETTINGS;
 use crate::window::KeyboardSettings;
 
+enum KeyOrIme {
```

Comment:
Maybe this should be named in some generic way, like KeyboardEvent, or InputEvent?